packages/cli/config/index.ts

```diff
@@ -332,13 +332,43 @@ const config = convict({
 				default: '',
 				env: 'N8N_JWT_AUTH_HEADER',
 				doc: 'The request header containing a signed JWT'
+      },
+      jwtHeaderValuePrefix: {
+				format: String,
+				default: '',
+				env: 'N8N_JWT_AUTH_HEADER_VALUE_PREFIX',
+				doc: 'The request header value prefix to strip (optional)'
 			},
 			jwksUri: {
 				format: String,
 				default: '',
 				env: 'N8N_JWKS_URI',
-				doc: 'The URI to fetch JWK Set for JWT auh'
-			},
+				doc: 'The URI to fetch JWK Set for JWT authentication'
+      },
+      jwtIssuer: {
+        format: String,
+				default: '',
+				env: 'N8N_JWT_ISSUER',
+				doc: 'JWT issuer to expect (optional)'
+      },
+      jwtNamespace: {
+        format: String,
+				default: '',
+				env: 'N8N_JWT_NAMESPACE',
+				doc: 'JWT namespace to expect (optional)'
+      },
+      jwtAllowedTenantKey: {
+        format: String,
+				default: '',
```

Comment:
Can you please fix the formatting in this block? 🙂

Reply:
Done.